source/scenes/Home/index.jsx

```diff
@@ -64,20 +64,27 @@ const Home = () => (
             hackathon. Students work with experienced open source mentors over 36 hours to
             contribute to open source through new features, bug fixes, and documentation changes.
           </p>
-          <h2>The Event.</h2>
+          <h2>The Event</h2>
           <p>
             This year, only groups who work with a mentor on an open source contribution and add a
             license will be eligible to win prizes. Contribute to a useful open source tool that you
             use, an open source project of your own, or join a team led by an open source mentor.
           </p>
           <h2>Join Us</h2>
           <p>
-            Interested in contributing to open source? Be sure to apply for your chance to attend.
-            Really want to come? Complete our Open Source Challenge for guaranteed admission to
-            HackIllinois 2019. It’s simple—just make one PR to any project between December 1, 2018
-            and the close of applications. New to open source? No problem. Our workshops and mentors
-            will help guide you through the intricacies of open source, from licensing to making
-            your first PR (pull request).
+            <b>Interested in contributing to open source?</b> Be sure to apply for your chance to
```

Comment:
We should not do this - Sophia does not like how this looks. Lets keep it one paragraph